src/Microsoft.ML.Data/Transforms/ConversionsExtensionsCatalog.cs

```diff
@@ -100,7 +106,7 @@ internal static TypeConvertingEstimator ConvertType(this TransformsCatalog.Conve
         /// <example>
         /// <format type="text/markdown">
         /// <![CDATA[
-        ///  [!code-csharp[KeyToValueMappingEstimator](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/ValueMappingStringToKeyType.cs)]
+        ///  [!code-csharp[KeyToValueMappingEstimator](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Conversion/ValueMappingStringToKeyType.cs)]
```

Comment:
>KeyToValueMappingEstimator [](start = 27, length = 26)

It should be the name of the extension method e.g. `MapKeyToValue`. Can you please also check this inconsistency in the rest of the file.